apps/wallet/src/shared/plausible.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import Plausible from 'plausible-tracker';
+
+// NOTE: The url of Sui wallet Chrome extension:
+// https://chrome.google.com/webstore/detail/sui-wallet/opcgpfmipidbgpenhmajoajpbobppdil
+const WALLET_URL = 'chrome-extension://opcgpfmipidbgpenhmajoajpbobppdil';
+
+const PLAUSIBLE_ENABLED = process.env.NODE_ENV !== 'development';
+
+const plausible = Plausible({
+    domain: WALLET_URL,
+});
+
+if (PLAUSIBLE_ENABLED && typeof document !== 'undefined') {
```

Comment:
Why do we need the document check?

Reply:
I'll add a comment, but basically the outbound link tracking works by inspecting the document, which doesn't exist in every context